app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, foreign_key: 'Usre'
```

Comment:
- [ ] A through association should have the alias model **refer** to actual model (_**User**_) using `class_name`, instead of `foreign_key`. As per the given _ERD_, the _Post_ **association** should have _Post_ belong to _User_ **through** _Author_, an alias of _User_, like below:
`belongs_to :author, class_name: 'User'` 

    Kindly ensure to improve upon this aspect.

Reply:
Thank you